src/data/services/StudioApiService.js

```diff
@@ -8,11 +8,25 @@ ensureConfig([
 const studioBaseUrl = getConfig().STUDIO_BASE_URL;
 
 class StudioApiService {
+  static getServiceUrl(courseID) {
+    return `${studioBaseUrl}/api/contentstore/v1/proctored_exam_settings/${courseID}`;
+  }
+
   static getProctoredExamSettingsData(courseID) {
     const apiClient = getAuthenticatedHttpClient();
-    const url = `${studioBaseUrl}/api/contentstore/v1/proctored_exam_settings/${courseID}`;
+    const url = StudioApiService.getServiceUrl(courseID);
     return apiClient.get(url);
   }
+
+  static saveProctoredExamSettingsData(courseID, dataToSave) {
+    const apiClient = getAuthenticatedHttpClient();
+    const url = StudioApiService.getServiceUrl(courseID);
+    return apiClient.post(url, dataToSave);
+  }
+
+  static getStudioUrl(courseID) {
```

Comment:
Nit: I'd rename this to something like `getStudioCourseRunUrl`. I'd expect `getStudioUrl` to return `studioBaseUrl`.